src/Microsoft.Data/DataFrameTable.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Data
+{
+    public class DataFrameTable
```

Comment:
What's the difference between a `DataFrame` and a `DataFrameTable`?

Maybe it would be good to have high-level class summary comments describing each public class.  You can write them like this:


```
    /// <summary>
    /// High level summary comment
    /// </summary>
   public class Foo
```

Reply:
A DataFrameTable is just a convenient store to hold the columns. The `DataFrame` class uses it to manipulate the columns and implement its algorithms.